models/balancer/ethereum/balancer_ethereum_sources.yml

```diff
@@ -321,6 +321,21 @@ sources:
           - name: contract_address
             description: ''
           - *token
+      
+      - name: BPool_call_setSwapFee
+        description: >
+          Decoded table related to the Balancer BPool contract.
+        loaded_at_field: call_block_time
+        columns:
+          - *call_tx_hash
+          - name: contract_address
+            description: ''
+          - name: swapFee
+            description: ''
```

Comment:
It seems me that the information is not accessible on the Dune app as none of the queries regarding table and/or schema information output it. Maybe it is just in the visible in the Spellbook repo. It would make sense that the information can be queried or is accessible in other places but who knows.

Reply:
there is a dbt docs page generated off these yaml files, but our dbt project has grown so large that it fails the docs generation command. we have a lower priority task to fix this and it's in progress. it won't be directly on the dune app, but we can hopefully add a link to it there for easy access.